src/Sylius/Bundle/CoreBundle/Migrations/Version20231103004216.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231103004216 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Making the uniqueness of the password_reset_token and email_verification_token a database constraint';
```

Comment:
Looking at recent migrations (As well as what I usually do), description should be similar in style to commit message.

For example,
```suggestion
        return 'Add unique indices for password_reset_token, email_verification_token fields to sylius_shop_user and sylius_admin_user tables';
```